src/open_raw.c

```diff
@@ -201,6 +199,7 @@ WavpackContext *WavpackOpenRawDecoder (
                 return NULL;
             }
 
+            WavpackHeader *wphdr = malloc (sizeof (WavpackHeader));
             memset (wphdr, 0, sizeof (WavpackHeader));
```

Comment:
Putting declarations in the middle of a block is C99, and not universally portable. Specifically, it doesn't work with Visual Studio 2008, which I still use.